cmake/modules/AddSwift.cmake

```diff
@@ -575,10 +648,18 @@ function(add_swift_host_tool executable)
     set_target_properties(${executable} PROPERTIES
       JOB_POOL_LINK swift_link_job_pool)
   endif()
-  if(${SWIFT_HOST_VARIANT_SDK} IN_LIST SWIFT_APPLE_PLATFORMS)
-    set_target_properties(${executable} PROPERTIES
-      BUILD_WITH_INSTALL_RPATH YES
-      INSTALL_RPATH "@executable_path/../lib/swift/${SWIFT_SDK_${SWIFT_HOST_VARIANT_SDK}_LIB_SUBDIR}")
+
+  if (ASHT_LIBSWIFT)
```

Comment:
I think that if someone wants to link against libswift, they should be able to just check if libswift the target exists and if so, link against it. Not something deep in the bowels here.